docker-compose.yml

```diff
@@ -52,13 +52,28 @@ services:
       dockerfile: Dockerfile.centos
     env_file:
       - ./conf/common.env
-      - conf/manager.env
+      - ./conf/manager.env
     command: ./manager/entrypoint.sh
     ports:
       - 8080:8080
     depends_on:
       - db
       - platform
 
+  vmaas_sync:
+    build:
+      context: .
+      dockerfile: Dockerfile.centos
+    env_file:
+      - ./conf/common.env
+      - ./conf/vmaas_sync.env
+    command: ./vmaas_sync/entrypoint.sh
+    ports:
+      # Reserving 8082 for evaluator :)
+      - 8083:8083
+    depends_on:
+      - db
+      - platform
```

Comment:
Component starts on `docker-compose up`, but how can I test that it's working? Could you add any testing scripts and description to README, please? I can not access :8083/metrics endpoint which should be exposed. Does it work for you?

Reply:
There was issue with connecting 2 local docker-compose networks. Should be fixed now.